internal/eval/eval.go

```diff
@@ -4723,98 +4723,98 @@ func parseJSONStructure(jsonData interface{}, nested bool) (resp []any) {
+// 	if len(args) < 3 || len(args)%2 == 0 {
+// 		return diceerrors.NewErrArity("ZADD")
+// 	}
 
-	key := args[0]
-	obj := store.Get(key)
+// 	key := args[0]
+// 	obj := store.Get(key)
 
-	var ss *sortedset.Set
+// 	var ss *sortedset.Set
 
-	if obj != nil {
-		var err []byte
-		ss, err = sortedset.FromObject(obj)
-		if err != nil {
-			return err
-		}
-	} else {
-		ss = sortedset.New()
-	}
+// 	if obj != nil {
+// 		var err []byte
+// 		ss, err = sortedset.FromObject(obj)
+// 		if err != nil {
+// 			return err
+// 		}
+// 	} else {
+// 		ss = sortedset.New()
+// 	}
 
-	added := 0
-	for i := 1; i < len(args); i += 2 {
-		scoreStr := args[i]
-		member := args[i+1]
+// 	added := 0
+// 	for i := 1; i < len(args); i += 2 {
+// 		scoreStr := args[i]
+// 		member := args[i+1]
 
-		score, err := strconv.ParseFloat(scoreStr, 64)
-		if err != nil || math.IsNaN(score) {
-			return diceerrors.NewErrWithMessage(diceerrors.InvalidFloatErr)
-		}
+// 		score, err := strconv.ParseFloat(scoreStr, 64)
+// 		if err != nil || math.IsNaN(score) {
+// 			return diceerrors.NewErrWithMessage(diceerrors.InvalidFloatErr)
+// 		}
 
-		wasInserted := ss.Upsert(score, member)
+// 		wasInserted := ss.Upsert(score, member)
 
-		if wasInserted {
-			added += 1
-		}
-	}
+// 		if wasInserted {
+// 			added += 1
+// 		}
+// 	}
 
-	obj = store.NewObj(ss, -1, object.ObjTypeSortedSet, object.ObjEncodingBTree)
-	store.Put(key, obj, dstore.WithPutCmd(dstore.ZAdd))
+// 	obj = store.NewObj(ss, -1, object.ObjTypeSortedSet, object.ObjEncodingBTree)
+// 	store.Put(key, obj, dstore.WithPutCmd(dstore.ZAdd))
 
-	return clientio.Encode(added, false)
-}
+// 	return clientio.Encode(added, false)
+// }
 
 // evalZRANGE returns the specified range of elements in the sorted set stored at key.
 // The elements are considered to be ordered from the lowest to the highest score.
-func evalZRANGE(args []string, store *dstore.Store) []byte {
-	if len(args) < 3 {
-		return diceerrors.NewErrArity("ZRANGE")
-	}
-
-	key := args[0]
-	startStr := args[1]
-	stopStr := args[2]
-
-	withScores := false
-	reverse := false
-	for i := 3; i < len(args); i++ {
-		arg := strings.ToUpper(args[i])
-		if arg == WithScores {
-			withScores = true
-		} else if arg == REV {
-			reverse = true
-		} else {
-			return diceerrors.NewErrWithMessage(diceerrors.SyntaxErr)
-		}
-	}
-
-	start, err := strconv.Atoi(startStr)
-	if err != nil {
-		return diceerrors.NewErrWithMessage(diceerrors.InvalidIntErr)
-	}
-
-	stop, err := strconv.Atoi(stopStr)
-	if err != nil {
-		return diceerrors.NewErrWithMessage(diceerrors.InvalidIntErr)
-	}
-
-	obj := store.Get(key)
-	if obj == nil {
-		return clientio.Encode([]string{}, false)
-	}
-
-	ss, errMsg := sortedset.FromObject(obj)
-
-	if errMsg != nil {
-		return diceerrors.NewErrWithMessage(diceerrors.WrongTypeErr)
-	}
-
-	result := ss.GetRange(start, stop, withScores, reverse)
-
-	return clientio.Encode(result, false)
-}
+// func evalZRANGEOld(args []string, store *dstore.Store) []byte {
```

Comment:
Please remove the old eval function (commented code)

Reply:
Removed